apl-core/src/main/java/com/apollocurrency/aplwallet/apl/eth/service/EthereumWalletService.java

```diff
@@ -220,6 +221,15 @@ public String sendApproveTransaction(EthWalletKey ethWalletKey, String spenderAd
         return sendApproveTransaction(ethWalletKey.getCredentials(), spenderAddress, value,  ethGasInfo.getAverageSpeedPrice());
     }
 
+
+    public EthBlock.Block getLastBlock() throws AplException.ExecutiveProcessException {
```

Comment:
Looks like ethBlockNumber do the same

Reply:
in this case I need block but not a number of the block.